src/lib/cli_lib/flag.ml

```diff
@@ -43,7 +43,7 @@ module Port = struct
       "Client to daemon rest server"
 
   let archive =
-    create ~name:"archive-port" ~default:default_archive
+    create ~name:"new-archive" ~default:default_archive
```

Comment:
by default if you run with `--archive` it will still use the old RocksDB one?

Reply:
yes